doc/architecture/database_schemata.md

```diff
@@ -149,3 +203,58 @@ subfamily }|--|| family : ""
+| **is_inactive**                  | false                            |
+| **last_visit**                   | 2023-04-04                       |
+| **honors**                       | 0                                | | 0 to infinity
+| **visits**                       | 0                                | | 0 to infinity
+| **harvested**                    | 0                                | | 0 to infinity
+| **creation_date**                | 2023-04-04                       |
+| **deletion_date**                | 2023-04-04                       |
+| **zoom_factor**                  | 100                              | | value used in formula "X by X cm", e.g. 100 would mean "100 x 100 cm", range from 10 to 100000
+| **background**                   | NULL                             |
+| **background_scale**             | NULL                             |
+| **geo_data**                     | NULL                             | | PostGis Geodata
+| **epsg_code**                    | NULL                             | | PostGis EPSG Code
+
+## `Users`
+
+| **_Column name_**                | **_Example_**                    | **_Initial rule_**                                                                                | **_Description_**                  |
+| :------------------------------- | :------------------------------- | :------------------------------------------------------------------------------------------------ | :--------------------------------- |
+| **id**                           | 1                                |
+| **nc_uid**                       | 1                                | | Nextcloud ID
+| **free_until**                   | 2023-04-04                       |
+| **app_language**                 | English                          |
+| **member_since**                 | 2023-04-04                       |
+| **experience**                   | {beginner, hobbyist, expert}     |
+| **preferences**                  | raised vegetable beds            |
+| **location**                     | Vienna, Austria                  |
+| **membership_points**            | 0                                | | 0 to infinity
+
+## `Blossoms`
+
+| **_Column name_**                | **_Example_**                    | **_Initial rule_**                                                                                | **_Description_**                  |
+| :------------------------------- | :------------------------------- | :------------------------------------------------------------------------------------------------ | :--------------------------------- |
+| **id**                           | 1                                |
+| **title**                        | Novice Gardener                  |
+| **description**                  | Plant your first plant           |
+| **condition**                    | plants.count() >= 1              | | condition used to check if milestone is reached
```

Comment:
Is this JavaScript?

Reply:
Not yet specified as it relies on where we want to check the conditions for the gain blossoms use case. It would make sense to check it in the backend but I have to look into Rust to understand in which format we would have to save these conditions.

Reply:
Please describe this shortly how you want to do it before you start implementing. We should keep this simple.